clusters/build-clusters/01_cluster/ci-search/deploy.yaml

```diff
@@ -4,164 +4,70 @@ metadata:
   name: ci-search
 
 ---
-# Import the ImageStream from CI
-kind: ImageStream
-apiVersion: image.openshift.io/v1
+apiVersion: route.openshift.io/v1
+kind: Route
 metadata:
   name: ci-search
-  namespace: ci-search
 spec:
-  lookupPolicy:
-    local: true
-  tags:
-  - from:
-      kind: DockerImage
-      name: registry.ci.openshift.org/ci/ci-search:latest
-    importPolicy:
-      scheduled: true
-    name: latest
-    referencePolicy:
-      type: Local
+  tls:
+    insecureEdgeTerminationPolicy: Redirect
+    termination: edge
+  to:
+    kind: Service
+    name: ci-search-redirect
 
-# deploy the controller
 ---
-kind: Service
 apiVersion: v1
+data:
+  default.conf: "server {\n  listen 8080;\n  return 302 https://search.dptools.openshift.org$request_uri;\n}\n"
+kind: ConfigMap
 metadata:
-  name: search
-  namespace: ci-search
-spec:
-  selector:
-    app: search
-  ports:
-  - port: 80
-    targetPort: 8080
+  name: ci-search-redirect-config
```

Comment:
nit... Can you move these up under `apiVersion`?

Reply:
added in https://github.com/openshift/release/pull/49035/commits/5e4f750c8426837cf7faa0ebf254941c0262beff